flink-state-backends/flink-statebackend-forst/src/main/java/org/apache/flink/state/forst/fs/ForStFlinkFileSystem.java

```diff
@@ -119,7 +121,11 @@ public static FileBasedCache getFileBasedCache(
                             new File(cacheBase.toString()), cacheReservedSize, SST_FILE_SIZE);
         }
         return new FileBasedCache(
-                Integer.MAX_VALUE, cacheLimitPolicy, cacheBase.getFileSystem(), cacheBase);
+                Integer.MAX_VALUE,
+                cacheLimitPolicy,
+                cacheBase.getFileSystem(),
```

Comment:
do we need to pass cacheBase.getFileSystem() when we already pass cacheBase?

Reply:
I will remove `cacheBase.getFileSystem()`.